deploy-board/deploy_board/templates/groups/group_details.html

```diff
@@ -37,12 +37,14 @@ <h4 class="panel-title pull-left">Groups</h4>
         </a>
     </div>
     {% endif %}
+    {% if group_info == None %}
```

Comment:
is `group_info` a valid value only when the group is an ASG group?

Reply:
right, non-ASG it is none.